test/functional/p2p_tx_download.py

```diff
@@ -284,17 +284,22 @@ def run_test(self):
 
         # Run each test against new bitcoind instances, as setting mocktimes has long-term effects on when
         # the next trickle relay event happens.
-        for test in [self.test_in_flight_max, self.test_inv_block, self.test_tx_requests,
-                     self.test_rejects_filter_reset]:
+        for test, with_inbounds in [
+            (self.test_in_flight_max, True),
+            (self.test_inv_block, True),
+            (self.test_tx_requests, True),
+            (self.test_rejects_filter_reset, False),
```

Comment:
> Should I just put a comment here # Avoid inbounds because they are not needed and slow down the test?

not really, doesn't add anything, so leave it as is